schema.sql

```diff
@@ -43,40 +43,10 @@ CREATE TABLE visits (
   visit_date DATE NOT NULL
 );
 
-INSERT INTO vets (name, age, date_of_graduation) VALUES ('William Tatcher', 45, '2000-04-23');
-INSERT INTO vets (name, age, date_of_graduation) VALUES ('Maisy Smith', 26, '2019-01-17');
-INSERT INTO vets (name, age, date_of_graduation) VALUES ('Stephanie Mendez', 64, '1981-05-04');
-INSERT INTO vets (name, age, date_of_graduation) VALUES ('Jack Harkness', 38, '2008-06-08');
+-- To optimize the query SELECT COUNT(*) FROM visits where animal_id = 4;
+CREATE INDEX animal_id_index ON visits (animal_id);
+-- to optimize the query SELECT * FROM visits where vet_id = 2;
+CREATE INDEX vet_id_index ON visits (vet_id);
+-- To optimize the query SELECT * FROM owners where email = 'owner_18327@mail.com';
+CREATE INDEX email_index ON owners (email);
```

Comment:
[OPTIONAL]
- [ ] This is a nice-to-have a suggestion as it won't improve the query that was reported as problematic in the requirements for this project.

As email addresses are not case sensitive, we might need to look for the emails in a case insensitive way. There is an option to reflect that in the index by using:

```JS
CREATE INDEX ON owners ((lower(email)));
```